lib/rucio/api/did.py

```diff
@@ -379,6 +379,22 @@ def get_metadata(scope, name):
     return api_update_return_dict(d)
 
 
+def bulk_get_metadata(dids, session=None):
+    """
+    Get metadata for a list of dids
+
+    :param dids: A list of dids.
+    :param session: The database session in use.
+    """
+
+    validate_schema(name='dids', obj=dids)
+    for entry in dids:
```

Comment:
i don't think this does what you intend? the entry variable won't be persisted back to dids, won't it?

Reply:
```
>>> dids = [{'scope': 'AA'}, {'scope': 'BB'}]
>>> for entry in dids:
...     entry['scope'] = 'blah'
... 
>>> dids
[{'scope': 'blah'}, {'scope': 'blah'}]
```